.travis.yml

```diff
@@ -4,7 +4,8 @@ php:
   - '5.6'
   - '7.0'
```

Comment:
There is an Separate [PR](https://github.com/Werkspot/FacebookAdsBundle/pull/6) for PHP7